modules/test/playwright/fixtures/documentLibraryPages.ts

```diff
@@ -10,7 +10,7 @@ import {test} from '@playwright/test';
 import {DocumentLibraryPage} from '../pages/document-library-web/documentLibrary.page';
 import {DocumentLibraryEditFolderPage} from '../pages/document-library-web/documentLibraryEditFolder.page';
 
-const documentLibraryPagesTest = test.extend<{
+const documentLibraryPages = test.extend<{
```

Comment:
Question: this is the new standard?

Reply:
No, it is just for consistency with other fixtures...